jscomp/ml/typedecl.ml

```diff
@@ -425,12 +425,18 @@ let transl_declaration ~typeRecordAsObject env sdecl id =
                 else typ in
               {lbl with  pld_type = typ }) in
           let lbls, lbls' = transl_labels env true lbls in
-          let lbls_opt = match lbls, lbls' with
-            | {ld_name = {txt = "..."}; ld_type} :: _, _ :: _ ->
+          let has_spread =
+            lbls
+            |> List.exists (fun l ->
+                   match l with
+                   | {ld_name = {txt = "..."}} -> true
```

Comment:
Does it make sense to have the treatment of spread somewhere (e.g. dedicated file)? E.g. no mention of "..." outside that file, or the has_spread functionality.
It might, or might not, be useful. What do you think?

Reply:
I think it might be useful, yeah. Might make sense to do it together with the next "feature" for type spreads, should we decide to do something like support type arguments.